vale/styles/RTD/Branding.yml

```diff
@@ -0,0 +1,6 @@
+extends: substitution
+message: Use '%s' instead of '%s'
+level: error
+swap:
+  Read The Docs: Read the Docs
+  RTD: Read the Docs
```

Comment:
I think we want to enable this usage in some places, but the above rule makes sense to me.

Reply:
It looks like we can disable and enable with the following markup comments:
```rst
Read The Docs

.. vale off

Read The Docs

.. vale on

Read The Docs
```

So in that example, an error would be thrown for text before `.. vale off` and for text after `.. vale on`.

Source: https://errata-ai.gitbook.io/vale/getting-started/markup#markup-based-configuration